content/en/docs/api/build.md

```diff
@@ -144,6 +148,24 @@ spec:
     contextDir: docker-build
 ```
 
+Example of a `Build` that specifies environment variables:
+
+```yaml
+apiVersion: shipwright.io/v1alpha1
+kind: Build
+metadata:
+  name: buildah-golang-build
+spec:
+  source:
+    url: https://github.com/shipwright-io/sample-go
+    contextDir: docker-build
+  env:
+    - name: EXAMPLE_VAR_1
+      value: "example-value-1"
+    - name: EXAMPLE_VAR_2
+      value: "example-value-2"
```

Comment:
aside from the where you initially specify content comment from @adambkaplan - my comment in the shipwright-io/build PR about the type of this field would come into play

if this field is going to be k8s corev1.EnvVar, then we need more examples to show getting content from configmaps, secrets, etc.

or our doc cites those are not supported

if, however, we go with a a simpler type that is a subset of corev1.EnvVar, then these examples seem sufficient

Reply:
Moved documentation to https://github.com/shipwright-io/build/pull/817